ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -658,6 +699,24 @@ S3FlushFinish(WT_STORAGE_SOURCE *storage, WT_SESSION *session, WT_FILE_SYSTEM *f
     return (ret);
 }
 
+/*
+ * S3ShowStatistics --
+ *     Log collected statistics.
+ */
+static int
+S3ShowStatistics(S3_STORAGE *s3)
+{
+    std::cout << "S3 list objects count: " << s3->statistics->listObjectsCount << std::endl;
+    std::cout << "S3 put object count: " << s3->statistics->putObjectCount << std::endl;
+    std::cout << "S3 get object count: " << s3->statistics->putObjectCount << std::endl;
+    std::cout << "S3 object exists count: " << s3->statistics->objectExistsCount << std::endl;
+
+    std::cout << "Non read/write file handle operations: " << s3->statistics->fhOps << std::endl;
+    std::cout << "File handle read operations: " << s3->statistics->fhReadOps << std::endl;
+
+    return (0);
```

Comment:
It doesnt appear the function can fail, or return anything other than 0, we can make the return type as void.